packages/pwa-kit-react-sdk/src/ssr/universal/hooks/index.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {useContext} from 'react'
+import {PageRequestContext, PageResponseContext} from '../contexts'
+
+export const usePageRequest = () => useContext(PageRequestContext)
+export const usePageResponse = () => useContext(PageResponseContext)
```

Comment:
Since this is a public api, I think it's best that you have some jsdoc comments on here to explain the eccentricities of these hooks (e.g. the behaviour on client vs server)

Reply:
If you are keen you can add a sample usage in the comment as well.